spark/src/main/scala/org/apache/spark/sql/delta/GeneratedColumn.scala

```diff
@@ -27,7 +27,11 @@ import org.apache.spark.sql.delta.sources.DeltaSourceUtils.GENERATION_EXPRESSION
 import org.apache.spark.sql.delta.sources.DeltaSQLConf
 import org.apache.spark.sql.delta.util.AnalysisHelper
 
-import org.apache.spark.sql.{AnalysisException, Column, Dataset, SparkSession}
+import org.apache.spark.sql.AnalysisException
```

Comment:
Can we do:

```scala
import org.apache.spark.sql.{AnalysisException, Column, Dataset, SparkSession}
import org.apache.spark.sql.ColumnImplicitsShim._
```

Reply:
it will cause scalastyle issues due to import ordering.